src/dimensions.jl

```diff
@@ -46,14 +46,14 @@ end
 end
 
 function virtual_call(::typeof(extent), ctx, start, stop)
-    if isconstant(start) && isconstant(stop)
+    #if isconstant(start) && isconstant(stop)
```

Comment:
I sorta think we need these checks. Was there a bug that led you to remove these?

Reply:
I made one last change that solves this in a slightly more direct way.